### DISCUSSION TITLE
How to get site_admin scope for OAuth App?

### DISCUSSION THREAD

**Discussion by fei0x** [community/community#24467] [Enterprise]
<p>Hello,</p>
<p>I been looking at the documentation for enterprise admin endpoint:</p>
<p>here: <a href="https://developer.github.com/enterprise/2.16/apps/building-oauth-apps/understanding-scopes-for-oauth-apps/#available-scopes" rel="nofollow noopener">understanding-scopes-for-oauth-apps</a></p>
<p>and here: <a href="https://developer.github.com/enterprise/2.16/v3/enterprise-admin/" rel="nofollow noopener">enterprise-admin</a></p>
<p>It sounds like I should be able to get the site_admin scope when trying to get an access token with an OAuth app, but I can’t seem to get it to work.</p>
<p>Here is an example of what I think should work:</p>
<pre><code>https://githubtrial.mysite.com/login/oauth/authorize?client_id=&lt;my-app-id&gt;&amp;redirect_uri=&lt;my-redirect-url&gt;&amp;scope=site_admin
</code></pre>
<p>(I also tried it with the ‘:’ as shown on the second link)</p>
<p>The Authorization screen does not ask me to confirm the scope, and upon retrieving my access token it doesn’t report the site_admin scope. I have got other scopes working, and I have been able to access the enterprise endpoint with a Personal Access Token. </p>
<p>If anyone knows how to do this it would be much appreciated.</p>

**Comment by nickvanw** [comment, marked as answer]
<p>Hey <a class="mention" href="/fei0x">@fei0x</a>!</p>
<p>I’m not sure about this off the top of my head, would you mind writing into <a href="https://github.com/contact" rel="nofollow noopener">https://github.com/contact</a> with your question, where we can route it to someone that can be more helpful than myself?</p>
<p>Thanks!</p>

**Comment by fei0x** [comment]
<p>will do. thanks.</p>
